src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
You should always include a fallback font in your `font-family` property. This is a good practice and ensures that if, for any reason, the browser can't load or find your first choice font, it will use the next available font in the list. For example, you could use a generic font family as a fallback: `font-family: Roboto, Arial, sans-serif, Helvetica;`